tests/SprykerTest/ChecksumGenerator/_support/ChecksumGeneratorTester.php

```diff
@@ -25,7 +25,7 @@
  * @method void comment($description)
  * @method \Codeception\Lib\Friend haveFriend($name, $actorClass = NULL)
  *
- * @SuppressWarnings(PHPMD)
+ * @SuppressWarnings(\SprykerTest\ChecksumGenerator\PHPMD)
```

Comment:
It should be \PHPMD\PHPMD, no?

Reply:
Our sniffer replaces it automatically, we have bunch of similar cases in the core, does it make sense?
<img width="1126" alt="Screenshot 2023-11-02 at 13 05 58" src="https://github.com/spryker/checksum-generator/assets/15832795/6d3eb193-c5bd-4584-8f25-d90f97572b5f">